nncf/onnx/quantization/quantize_model.py

```diff
@@ -44,6 +54,60 @@
 TTensor = TypeVar("TTensor")
 
 
+def check_model_protobuf_size(model: onnx.ModelProto) -> None:
+    """
+    Checks whether the serialized ONNX model exceeds the 2GB protobuf size limit.
+    If the size limit is exceeded, a `nncf.ValidationError` is raised.
+
+    :param model: The ONNX model to be checked.
+    """
+    MAXIMUM_PROTOBUF = 2000000000  # Limitation of single protobuf file is 2GB
```

Comment:
Why do you use capital letters to create a constant in a function? Why do you use 2000000000  instead of 2^31?

Reply:
I've followed the ONNX approach to handle it.

https://github.com/onnx/onnx/blob/main/onnx/checker.py#L176-L179

Reply:
Pay attention on size of constant: https://github.com/onnx/onnx/blob/94e820742ea3f9716a62d486611e46c61355d7f5/onnx/checker.py#L47

Reply:
Done